### DISCUSSION TITLE
Difference between izzy and play versions?

### DISCUSSION THREAD

**Discussion by Ecasept** [deckerst/aves#202] [Q&A] [answered]
There are two types of apks released, izzy and play. The play version is two times larger than the izzy one. Why, and what's the difference? Does it matter which one I pick?

**Comment by deckerst** [comment, marked as answer]
There are 2 differences:
- content: the Play version contains Crashlytics for optional error reporting, while the Izzy version does not include it,
- packaging: the Play version is a "fat" APK that contains code for all ABIs, while the Izzy APKs are split for each ABI (with only the code for that ABI, hence the difference in size).

If you think Crashlytics is evil (even when disabled), and you know what ABI matches your device, you should take the Izzy one. Otherwise the Play one is fine. The file is larger, but in the end on the device it's more or less the same thing.

But the easiest way is to *not* download the file here and to get it instead from the Play store or F-Droid (with the Izzy repository). By the way, the Play store dynamically optimizes the process so users actually only download the code needed for their specific device and locale.

I should write this in the README, thanks for the heads-up! And let me know if it's still unclear.

**Comment by deckerst** [comment]
App version table on the wiki is [here](https://github.com/deckerst/aves/wiki/App-Versions).
